### ISSUE TITLE
scanreply and scan difference

### ISSUE THREAD

**Issue by anoopswsib** [solita/clamav-rest#22]
Can you please explain why you have /scanreply end point in your code. Trying to understand the difference between scanreply and scan. Thanks

**Comment by iJebus** [comment]
I agree that documentation could be improved, but regardless, based on #13 it looks like `scanReply` should return the raw ClamAV result rather than a boolean.
